exercises/house/canonical-data.json

```diff
@@ -0,0 +1,322 @@
+            "that tossed the dog",
+            "that worried the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built."
+          ]
+        },
+        {
+          "description": "verse nine",
+          "property": "verse",
+          "verse": 9,
+          "expected": [
+            "This is the priest all shaven and shorn",
+            "that married the man all tattered and torn",
+            "that kissed the maiden all forlorn",
+            "that milked the cow with the crumpled horn",
+            "that tossed the dog",
+            "that worried the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built."
+          ]
+        },
+        {
+          "description": "verse 10",
+          "property": "verse",
+          "verse": 10,
+          "expected": [
+            "This is the rooster that crowed in the morn",
+            "that woke the priest all shaven and shorn",
+            "that married the man all tattered and torn",
+            "that kissed the maiden all forlorn",
+            "that milked the cow with the crumpled horn",
+            "that tossed the dog",
+            "that worried the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built."
+          ]
+        },
+        {
+          "description": "verse 11",
+          "property": "verse",
+          "verse": 11,
+          "expected": [
+            "This is the farmer sowing his corn",
+            "that kept the rooster that crowed in the morn",
+            "that woke the priest all shaven and shorn",
+            "that married the man all tattered and torn",
+            "that kissed the maiden all forlorn",
+            "that milked the cow with the crumpled horn",
+            "that tossed the dog",
+            "that worried the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built."
+          ]
+        },
+        {
+          "description": "verse 12",
+          "property": "verse",
+          "verse": 12,
+          "expected": [
+            "This is the horse and the hound and the horn",
+            "that belonged to the farmer sowing his corn",
+            "that kept the rooster that crowed in the morn",
+            "that woke the priest all shaven and shorn",
+            "that married the man all tattered and torn",
+            "that kissed the maiden all forlorn",
+            "that milked the cow with the crumpled horn",
+            "that tossed the dog",
+            "that worried the cat",
+            "that killed the rat",
+            "that ate the malt",
+            "that lay in the house that Jack built."
+          ]
+        },
+        {
+          "description": "multiple verses",
+          "property": "verse",
```

Comment:
I don't really care (so don't worry about waiting for my approval after this) but I am suspicious that the property is the same but the other keys are different ("verse" versus "start verse" + "end verse"). https://github.com/exercism/problem-specifications/blob/master/exercises/beer-song/canonical-data.json did not use the same property when the keys were different.